cargo-dist/src/lib.rs

```diff
@@ -755,6 +759,14 @@ pub(crate) fn copy_dir(src_path: &Utf8Path, dest_path: &Utf8Path) -> Result<()>
     Ok(())
 }
 
+pub(crate) fn copy_file_or_dir(src_path: &Utf8Path, dest_path: &Utf8Path) -> Result<()> {
```

Comment:
Wonder if we want this in `AxoAsset` at some point.

Reply:
Could be -- these shims are all weird artifacts of pre-axoasset